imsv-docs-astro/src/content/docs/guides/funding-protocols/custodial-funding-protocol.mdoc

```diff
@@ -175,76 +176,105 @@ content type is application/json.
   can be useful for debugging.
 
 ---
-* {% wrap %} cardId {% /wrap %}
-* {% wrap 9 %}9ae177ea85ed4128{% /wrap %}
-* Immersve card identifier.
-
----
-* {% wrap %} cardholderId {% /wrap %}
-* {% wrap 9 %}d44d045826d4484c{% /wrap %}
-* Immersve cardholder account id.
-
----
-* {% wrap %} fundingSourceId {% /wrap %}
-* {% wrap 9 %}c65fe4d184a24619{% /wrap %}
-* Immersve funding source identifier.
+* {% wrap %} paymentId {% /wrap %}
+* {% wrap 9 %}38fb58acc55b4907{% /wrap %}
+* Immersve payment transaction identifier. This attribute should be used to find
+  the existing payment resource to update accordingly based on the message type.
 
 ---
-* {% wrap %} fundingSourceExternalId {% /wrap %}
-* abc123
-* The client user reference stored on the Immersve funding source. This
-  attribute should be used to resolve the custodial ledger account to be debited.
+* {% wrap %} paymentFinalized {% /wrap %}
+* true
+* Flag indicating if the payment is expected to have further updates.
+  Messages of type "auth-request" will always be non-finalized. Payments are
+  eventually finalized after being cleared, expired or cancelled. Payments may
+  be updated after being finalized if, for example, the payment is reversed.
 
----
-* {% wrap %} paymentId {% /wrap %}
-* {% wrap 9 %}38fb58acc55b4907{% /wrap %}
-* Immersve payment transaction identifier.
 
 ---
 * {% wrap %} paymentAmount {% /wrap %}
 * 1312
-* Required payment amount in minor units.
+* Total payment amount that the cardholder will be debited in minor units. The
+  payment amount may increase or decrease when cleared or reversed. A refund
+  payment will have a negative amount.
 
 ---
 * {% wrap %} paymentCurrency {% /wrap %}
 * USD
 * Required payment currency.
 
 ---
-* {% wrap %} paymentAdjustment {% /wrap %}
+* {% wrap %} paymentAcquirerAmount {% /wrap %}
+* 978
+* Payment amount represented in the merchant's local currency.
 
 ---
-* {% wrap %} paymentFinalized {% /wrap %}
+* {% wrap %} paymentAcquirerCurrency {% /wrap %}
+* GBP
+* The merchant's local currency.
 
 ---
 * {% wrap %} paymentFeeAmount {% /wrap %}
+* 32
+* Immersve's payment fee amount in minor units. The fee currency is
+  the same as the payment currency.
 
 ---
-* {% wrap %} paymentFeeCurrency {% /wrap %}
```

Comment:
paymentFeeCurrency is listed in Notion but deleted here

Reply:
I removed from the design docs. It's useless because it's always the same as the payment currency.